language/tools/move-unit-test/src/lib.rs

```diff
@@ -145,6 +138,31 @@ impl UnitTestingConfig {
         test_plan.map(|tests| TestPlan::new(tests, files, units))
     }
 
+    /// Build a test plan from a unit test config
+    pub fn build_test_plan(&self) -> Option<TestPlan> {
+        let mut deps = self.dep_files.clone();
+
+        // We always need to have the unit testing module in otherwise the modules will fail to
+        // compile in test mode
+        if !self
+            .source_files
+            .iter()
+            .chain(deps.iter())
+            .any(|file_path| file_path.contains("UnitTest.move"))
```

Comment:
wat

What if someone else names a file "UnitTest.move"? whats this about?

Reply:
This is because we require the UnitTest module to be included in the deps in order to compile (and run) the unit tests. Although this is pretty fragile...

Let me see if I can figure out a good fix for this.

Reply:
I would just look for the module in the deps, and yell at them if its not there

Reply:
Or maybe just always insert it somehow and yell at them if they tried to redefine it?

Reply:
True, however then whenever someone wants to run unit tests they'll need to pass this as a dependency which seems painful. 

My thoughts to solve this would be to parse them first and if it's not there adding in the module instead of yelling at them?

Reply:
hmmm. Maybe we should require it to be there and then solve that concern with the build system/packages later?

Reply:
Agreed that long-term this is an issue that should/will be solved with a build/package system. However in the short term this feels terribly unergonomic :( 

What would you think of always including the unit test module, and yelling at the programmer if they include the `UnitTest` module? This feels more ergonomic at least for now.

Reply:
> What would you think of always including the unit test module, and yelling at the programmer if they include the UnitTest module? This feels more ergonomic at least for now.

Sorry for the spam but yeah I added that suggestion later. I think that seems like a decent short term solution

Reply:
Should be all fixed now :)